docs/deployment/locally-api.md

```diff
@@ -158,8 +158,11 @@ The application relies on the following environment variables to run:
 
 #### Database Environment Variables
 
-- `DATABASE_URL`: required by develop and production environments, should be a standard format database url e.g.
+- `DATABASE_URL`: (required) configure the database to connect to. Should be a standard format database url e.g.
   postgres://user:password@host:port/db_name
+- `REPLICA_DATABASE_URLS`: (optional) configure an optional number of read replicas. Should be a comma separated list of
+  standard format database urls. e.g.
+  postgres://user:password@replica1.db.host/flagsmith,postgres://user:password@replica2.db.host/flagsmith
```

Comment:
Do we know if commas are reserved characters in this URL scheme?

Reply:
Hmm... yes, good thinking. I'm not sure about elsewhere but I guess a comma is a perfectly reasonable character to include in a password. I'll have to test this out and see if it can be worked around.

Reply:
I've added a new environment variable `REPLICA_DATABASE_URLS_DELIMITER` so that people can configure the delimiter as needed. I can't foresee any other issues here?